packages/oui-pagination/src/pagination.html

```diff
@@ -0,0 +1,77 @@
+                <span ng-bind="$ctrl.getLastItemOffset()"></span></button>
+            <oui-dropdown-content>
+                <div role="menu" class="oui-pagination-menu">
+                    <div class="oui-pagination-menu__group"
+                        ng-bind="::$ctrl.config.translations.resultsPerPage"></div>
+                    <div class="oui-pagination-menu__items-list">
+                        <button type="button"
+                            ng-repeat="pageSize in $ctrl.pageSizeList track by $index"
+                            class="oui-pagination-menu__item"
+                            ng-disabled="pageSize === $ctrl.pageSize"
+                            ng-bind="pageSize"
+                            ng-click="$ctrl.onPageSizeChange(pageSize)"></button>
+                    </div>
+                </div>
+            </oui-dropdown-content>
+        </oui-dropdown>
+        <span ng-bind="$ctrl.translations.ofNResults"></span>
+    </div>
+
+    <div class="oui-pagination__selector">
+        <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+            type="button"
+            aria-label="Previous page"
+            ng-disabled="$ctrl.currentPage === 1"
+            ng-click="$ctrl.onPageChange($ctrl.getCurrentPage() - 1)">
+            <i class="oui-icon oui-icon-chevron-left" aria-hidden="true"></i>
+        </button>
+        <div class="oui-button-group"
+            ng-if="$ctrl.pageCount <= 5">
+            <button type="button"
+                class="oui-button oui-button_small-width"
+                ng-repeat="page in $ctrl.pageRange track by page"
+                ng-class="{
+                    'oui-button_primary oui-pagination-button_selected': page === $ctrl.currentPage,
+                    'oui-button_secondary': page != $ctrl.currentPage
+                }"
+                ng-disabled="page === $ctrl.currentPage"
+                ng-bind="page"
+                ng-click="$ctrl.onPageChange(page)"></button>
+        </div>
+        <oui-dropdown ng-if="$ctrl.pageCount > 5">
+            <button oui-dropdown-trigger
+                type="button"
+                ng-attr-aria-label="$ctrl.translations.currentPageOfPageCount"
+                class="oui-button oui-button_dropdown oui-dropdown__trigger">
+                <span ng-bind="$ctrl.translations.currentPageOfPageCount"></span>
+                <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+            </button>
+            <oui-dropdown-content>
+                <div role="menu" class="oui-pagination-menu">
+                    <div class="oui-pagination-menu__items-list">
+                        <button type="button"
+                            ng-repeat="page in $ctrl.pageRange track by page"
+                            class="oui-pagination-menu__item"
+                            ng-disabled="page === $ctrl.currentPage"
```

Comment:
(optional) I think we will need to add an `aria-label` on those buttons too otherwise the client will only understand the number instead of **page 2 of 5**.

Reply:
Fixed